client/components/nav-drawer.jsx

```diff
@@ -36,7 +36,7 @@ class Navigation extends React.Component {
             <i onClick={this.handleClick} className="fas fa-bars"></i>
          </div>
         <div className="nav-header row">
-            <h1 className="nav-title">Home</h1>
+            <h1 className="nav-title">{this.props.state.path}</h1>
```

Comment:
I don't know where this `props.state` thing is coming from, but it's confusing to name a prop `state`. How about just passing the `path`?